modules/server/src/server.js

```diff
@@ -5,9 +5,35 @@ import bodyParser from 'body-parser';
 import projectsRoutes from './projects';
 import { getProjects } from './utils/projects';
 import startProject, { getDefaultServerSideFilter } from './startProject';
-import { ES_HOST, PROJECT_ID, MAX_LIVE_VERSIONS } from './utils/config';
+import { ES_HOST, ES_USER, ES_PASS, ES_LOG, PROJECT_ID, MAX_LIVE_VERSIONS } from './utils/config';
 import { fetchProjects } from './projects/getProjects';
 
+export const buildClientViaEnv = () => {
+  return buildClient(ES_HOST, ES_USER, ES_PASS, ES_LOG)
+}
+
+export const buildClient = (esHost, esUser, esPass, esLog) => {
+  if (!esHost) {
+    console.error('no elasticsearch host was provided');
+  }
+
+  let esConfig = { 
+    node: esHost,
+    log: esLog
+  }
+
+  if (esUser){
+	  if (!esPass) {
+		  console.error('ES user was defined, but password was not');
+	  }
+	  esConfig['auth'] = {
+		  username: esUser,
+		  password: esPass
+	  }
+  }
```

Comment:
there's some inconsistent indentation here... we normally use [prettier](https://prettier.io/) to handle code formatting as a commit hook, but I guess we never installed it for this repo